_includes/manuals/1.20/1.2_release_notes.md

```diff
@@ -376,7 +375,10 @@ Foreman should now run properly in FIPS-enabled environments.
 
 ### Deprecations
 
+* The ability to disable login, organizations or locations via settings has been deprecated and will be
+removed in version 1.21. Please enable these settings in your `settings.yaml` file.
```

Comment:
Can you elaborate what this effectively mean for users upgrading or installing 1.20? A warning messages in logs? I believe it's not enough and we should be more verbose, probably some persistent banner, they need to test it now, not after there is no way back. Any ideas?

Reply:
There is a notification of this change pointing to the discussion thread on discourse that will be displayed to all admins for a month after the upgrade.